composites/CoursesOverview/Card.js

```diff
@@ -55,10 +64,10 @@ class Card extends React.Component {
 
 		if ( this.props.header.link ) {
 			return (
-				<HeaderLink href={ this.props.header.link }>
+				<OutboundHeaderLink href={ this.props.header.link }>
```

Comment:
we should use `rel={ null }` to omit noopener and noreferrer, as these links point to yoast.com